src/Evolution/Systems/CurvedScalarWave/Equations.hpp

```diff
@@ -10,10 +10,12 @@
 #include "Utilities/Gsl.hpp"
 #include "Utilities/TMPL.hpp"
 
+/// \cond
 template <typename>
 class Variables;
 
 class DataVector;
+/// \endcond
```

Comment:
only cond templates or all forward declarations?

Reply:
I think our best bet is probably cond every forward declaration in header files, nothing in cpp files. This will be the simplest rule for people to remember. `cond`ing them doesn't have any adverse effects.